public/js/helpers/add_day.js

```diff
@@ -0,0 +1,8 @@
+var addDay = function (dataSet) {
+  console.log(addDay);
+  var plusDay = parseInt(dataSet.date.split('-')[2]) + 1;
+  var plusDate = dataSet.date.split('-');
+  plusDate.pop();
+  plusDate.push(plusDay.toString());
+  return plusDate;
```

Comment:
this doesn't work for last day of the month!

Reply:
![screen shot 2017-09-13 at 17 44 52](https://user-images.githubusercontent.com/16781318/30383832-4de22ac8-98ab-11e7-8c95-e9dca79be961.png)